src/crawlee/http_clients/_httpx.py

```diff
@@ -215,6 +223,16 @@ def _get_client(self, proxy_url: str | None) -> httpx.AsyncClient:
 
         return self._client_by_proxy_url[proxy_url]
 
+    def _get_headers(self, explicit_headers: HttpHeaders | None) -> HttpHeaders | None:
```

Comment:
The `get_` prefix doesn't seem fitting here - how about something like "combine"?